### PR TITLE
Add opt-out import and cclf-import workflows

### PR DESCRIPTION
**Pull request by austincanada**
## 🎫 Ticket

https://jira.cms.gov/browse/BCDA-7899

## 🛠 Changes

Added cclf-import workflows.
Added Sean's opt-out-import workflows.
Added unit testing workflow for cclf and opt-out workflow. 

## ℹ️ Context for reviewers

Creating a merge PR in order to test run the workflows. Some things will probably have to change within the workflows themselves.

## ✅ Acceptance Validation

Workflows completed:
Screenshot 2024-06-18 at 12 23 13 PM
Screenshot 2024-06-18 at 12 23 38 PM

Lambdas updated:
Screenshot 2024-06-18 at 12 29 49 PM

## 🔒 Security Implications

- [ ] This PR adds a new software dependency or dependencies.
- [ ] This PR modifies or invalidates one or more of our security controls.
- [ ] This PR stores or transmits data that was not stored or transmitted before.
- [ ] This PR requires additional review of its security implications for other reasons.

If any security implications apply, add Jason Ashbaugh (GitHub username: StewGoin) as a reviewer and do not merge this PR without his approval.

### CODE REVIEW

**Comment by gsf** [comment]
@austincanada With this PR up, can I close #950?

**Comment by austincanada** [comment]
> @austincanada With this PR up, can I close #950?

Yes, that's fine.

File: .github/workflows/opt-out-import-test-integration.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          role-to-assume: arn:aws:iam::${{ secrets.BFD_ACCOUNT_ID }}:role/bfd-test-eft-bcda-bucket-role
+          role-chaining: true
+          role-skip-session-tagging: true
+      - name: Upload test file to the BFD bucket to trigger lambda function via SNS message
+        run: |
+          pwd
+          aws s3 cp --no-progress ../shared_files/synthetic1800MedicareFiles/test/T\#EFT.ON.ACO.NGD1800.DPRF.D181120.T1000009 \
+            s3://bfd-test-eft/bfdeft01/bcda/in/T.NGD.DPC.RSP.D$(date +'%y%m%d').T$(date +'%H%M%S')1.IN
+
+  # TODO Check bucket for response file
```
Comment (gsf) [inline .github/workflows/opt-out-import-test-integration.yml:50]: See where I've figured out one way to check the response file: https://github.com/CMSgov/ab2d-lambdas/blob/ac828968f2f17c4440b0724bf196e8939b39c935/.github/workflows/opt-out-import-test-integration.yml#L49-L53

Comment (kyeah) [reply .github/workflows/opt-out-import-test-integration.yml:50]: We don't have response files for BCDA, so we'll just want to do the database check below if possible (or maybe just verify that the file was deleted from S3 after being ingested by the app code.)

Will need some arbitrary waiting period / loop or something to allow the lambda to run.

Comment (austincanada) [reply .github/workflows/opt-out-import-test-integration.yml:50]: Do we have any examples of us verifying the file was deleted or checking the database? Looking through ab2d and dpc there's no implementation that I can find to mirror them.

**Review by laurenkrugen-navapbc — APPROVED** [review]

File: .github/workflows/opt-out-import-test-integration.yml (hunk 1 above)
Comment (gsf) [reply .github/workflows/opt-out-import-test-integration.yml:50]: No response files? That does change things. Hmm agreed then that the best check would be a job in the workflow to run a self-hosted runner and check the database. Good future work to schedule

**Pull request merged**